xml/yast2_sw.xml

```diff
@@ -320,47 +320,49 @@
      repository remain installed.
     </para>
    </tip>
-   <sect3 os="sles;sled">
+   <tip os="sles;sled" xml:id="tip-yast-software-search-online">
```

Comment:
Suggestion to have this as plain procedure instead of tip

Reply:
The online search needs to be separated from the normal/offline search.

* We need at least a title and a sentence that explains the benefit of the online search (search all modules, even unregistered ones).
* It's SLE-only and needs to be profiled.

Introducing a `<sec3>` would work, but results in a lone section. We could then move the content of the normal/offline search to yet another `<sec3>`, but IHNMO that is too much overhead and too much nesting I think. The online search is an extra by definition, thus I think a `<tip>` is just right.